base/ltfssaxes.dtx

```diff
@@ -192,503 +250,3177 @@
+\DeclareFontSeriesChangeRule {sx}{ul}   {ulsx} {}
+\DeclareFontSeriesChangeRule {sx}{el}   {elsx} {}
+\DeclareFontSeriesChangeRule {sx}{l}    {lsx}  {}
+\DeclareFontSeriesChangeRule {sx}{sl}   {slsx} {}
+\DeclareFontSeriesChangeRule {sx}{sb}   {sbsx} {bsx}
+\DeclareFontSeriesChangeRule {sx}{b}    {bsx}  {}
+\DeclareFontSeriesChangeRule {sx}{eb}   {ebsx} {bsx}
+\DeclareFontSeriesChangeRule {sx}{ub}   {ubsx} {bsx}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {x}{ul}    {ulx}  {}
+\DeclareFontSeriesChangeRule {x}{el}    {elx}  {}
+\DeclareFontSeriesChangeRule {x}{l}     {lx}   {}
+\DeclareFontSeriesChangeRule {x}{sl}    {slx}  {}
+\DeclareFontSeriesChangeRule {x}{sb}    {sbx}  {bx}
+\DeclareFontSeriesChangeRule {x}{b}     {bx}   {}
+\DeclareFontSeriesChangeRule {x}{eb}    {ebx}  {bx}
+\DeclareFontSeriesChangeRule {x}{ub}    {ubx}  {bx}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ex}{ul}   {ulex} {}
+\DeclareFontSeriesChangeRule {ex}{el}   {elex} {}
+\DeclareFontSeriesChangeRule {ex}{l}    {lex}  {}
+\DeclareFontSeriesChangeRule {ex}{sl}   {slex} {}
+\DeclareFontSeriesChangeRule {ex}{sb}   {sbex} {bex}
+\DeclareFontSeriesChangeRule {ex}{b}    {bex}  {}
+\DeclareFontSeriesChangeRule {ex}{eb}   {ebex} {bex}
+\DeclareFontSeriesChangeRule {ex}{ub}   {ubex} {bex}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ux}{ul}   {ulux} {}
+\DeclareFontSeriesChangeRule {ux}{el}   {elux} {}
+\DeclareFontSeriesChangeRule {ux}{l}    {lux}  {}
+\DeclareFontSeriesChangeRule {ux}{sl}   {slux} {}
+\DeclareFontSeriesChangeRule {ux}{sb}   {sbux} {bux}
+\DeclareFontSeriesChangeRule {ux}{b}    {bux}  {}
+\DeclareFontSeriesChangeRule {ux}{eb}   {ebux} {bux}
+\DeclareFontSeriesChangeRule {ux}{ub}   {ubux} {bux}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {sbuc}{ul} {uluc} {}
+\DeclareFontSeriesChangeRule {sbuc}{el} {eluc} {}
+\DeclareFontSeriesChangeRule {sbuc}{l}  {luc}  {}
+\DeclareFontSeriesChangeRule {sbuc}{sl} {sluc} {}
+\DeclareFontSeriesChangeRule {sbuc}{sb} {sbuc} {}
+\DeclareFontSeriesChangeRule {sbuc}{b}  {buc}  {}
+\DeclareFontSeriesChangeRule {sbuc}{eb} {ebuc} {buc}
+\DeclareFontSeriesChangeRule {sbuc}{ub} {ubuc} {buc}
+\DeclareFontSeriesChangeRule {sbuc}{uc} {sbuc} {}
+\DeclareFontSeriesChangeRule {sbuc}{ec} {sbec} {}
+\DeclareFontSeriesChangeRule {sbuc}{c}  {sbc}  {}
+\DeclareFontSeriesChangeRule {sbuc}{sc} {sbsc} {}
+\DeclareFontSeriesChangeRule {sbuc}{sx} {sbsx} {}
+\DeclareFontSeriesChangeRule {sbuc}{x}  {sbx}  {}
+\DeclareFontSeriesChangeRule {sbuc}{ex} {sbex} {}
+\DeclareFontSeriesChangeRule {sbuc}{ux} {sbux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {sbec}{ul} {ulec} {}
+\DeclareFontSeriesChangeRule {sbec}{el} {elec} {}
+\DeclareFontSeriesChangeRule {sbec}{l}  {lec}  {}
+\DeclareFontSeriesChangeRule {sbec}{sl} {slec} {}
+\DeclareFontSeriesChangeRule {sbec}{sb} {sbec} {bec}
```

Comment:
```suggestion
\DeclareFontSeriesChangeRule {sbec}{sb} {sbec} {}
```

Another rule with a useless fallback because the third argument is the same as the first.